paasta_tools/cli/cmds/logs.py

```diff
@@ -328,16 +372,37 @@ def get_log_reader():
 
 
 class LogReader(object):
-    def __init__(self, **kwargs):
-        pass
+    def __init__(self):
+        # Tailing, i.e actively viewing logs as they come in
+        self.SUPPORTS_TAILING = False
```

Comment:
Sure they're not constants in a strict sense that they're only assigned once but semantically I'd argue they're still constants and deserve to be named as such.